src/acuvity/apexextend.py

```diff
@@ -78,7 +78,7 @@ def list_detectable_secrets(self) -> List[str]:
                     if detector.group == "Secrets"
                 ]
                 detectable_secrets.extend(detectable_secrets_local)
-        return sorted(detectable_secrets)
+        return sorted(set(detectable_secrets))
```

Comment:
if you want to return a set, then you must change the return type to set

Reply:
fixed.